.gitlab-ci.yml

```diff
@@ -95,22 +96,150 @@ whois-integration-test:
     - mariadb:10.2
   tags:
     - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
+      allow_failure: true
+  script:
+    - mvn -DskipTests install
+    - mvn $MAVEN_OPTS -Ddb.host=mariadb verify -Pintegration -pl whois-api -T 1C
+
+whois-client-integration-test:
+  <<: *it_env
+  services:
+    - mariadb:10.2
+  tags:
+    - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
```

Comment:
can we create a generic step so that we do not have to repeat the rules and all IT tests extends it?

Reply:
So for example if we change the mariadb version now we need to change at 7 diffeent places

Reply:
yes, you are right let me check of to do that